catalog/models.py

```diff
@@ -87,10 +95,23 @@ def get_queryset(self):
     def get_by_category(self, category: models.Model, ordering: [str]=None) -> models.QuerySet:
         return self.get_queryset().get_by_category(category, ordering)
 
+    def get_category_descendants(self, category: models.Model, ordering: [str]=None) -> models.QuerySet:
```

Comment:
`[str]=None] -> [str] = None`

Reply:
@artemiy312 , we assign default args without whitespaces by default: `ordering=None` instead of `ordering = None`